CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This change log adheres to standards from [Keep a CHANGELOG](http://keepachangelog.com).
 
 ## [Unreleased]
+## [2.13.1] - 2019-01-28
```

Comment:
version numbers aren't decided in PRs, typically :-)
```suggestion
```

Reply:
Ah, ok thanks! I will update to make it more generic